LibreNMS/Snmptrap/Handlers/AdvaAttributeChange.php

```diff
@@ -0,0 +1,239 @@
+<?php
+/**
+ * AdvaAtributeChange.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Attribute change traps show changes to Adva configuration values after they are submitted.
+ * This handler only catches some of those changes and aims to provide the user with
+ * information about what configuration module was modified.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 KanREN, Inc
+ * @author     Heath Barnhart <hbarnhart@kanren.net>
+ */
+
+namespace LibreNMS\Snmptrap\Handlers;
+
+use App\Models\Device;
+use LibreNMS\Interfaces\SnmptrapHandler;
+use LibreNMS\Snmptrap\Trap;
+use Log;
+
+class AdvaAttributeChange implements SnmptrapHandler
+{
+    /**
+     * Handle snmptrap.
+     * Data is pre-parsed and delivered as a Trap.
+     *
+     * @param Device $device
+     * @param Trap $trap
+     * @return void
+     */
+    public function handle(Device $device, Trap $trap)
```

Comment:
This function is very long and hard to read.  Moving some of the code to private functions would help a lot.  Like the first if statement calling:
```php
if ($trap->findOid('CM-SYSTEM-MIB::sysLog')) {
   $this->handleSyslog($trap);
} elseif ($trap->findOid('CM-SYSTEM-MIB::aclEntry')) {
```

Reply:
Will do.

Reply:
Ok, I'll go ahead and do that for all the sections. Some of the sections I pulled back the amount of information into a generic "this was modified message". Splitting out into more functions will allow me or someone else go back and make the traps more informative again if desired.